cwf/gateway/docker/go/Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN apt-get update && apt-get install -y \
     libjansson-dev \
     libjemalloc-dev \
     libssl-dev \
-    libsystemd-dev \
+    libsystemd-dev=245.4-4ubuntu3.18 \
```

Comment:
magma-preferences is not used in this Dockerfile, i.e., should not have a need to be pinned